src/azul/service/manifest_service.py

```diff
@@ -1005,8 +1005,8 @@ def source_filter(self) -> SourceFilters:
     @classmethod
     def command_lines(cls, url: str) -> JSON:
         return {
-            'cmd.exe': f'curl.exe --location {cls._cmd_exe_quote(url)} | curl.exe --config -',
-            'bash': f'curl --location {shlex.quote(url)} | curl --config -'
+            'cmd.exe': f'curl.exe --location -f {cls._cmd_exe_quote(url)} | curl.exe --config -',
```

Comment:
Explain the reasoning behind -f in a comment.

Reply:
… behind us using -f that is.

Reply:
I might have given contradicting guidance earlier but I think we should stick with long options. This aligns well with my request to switch to ' '.join([…]` for command line composition in another PR.